### PR TITLE
Issue #4586: Moved all resources for JavadocTagContinuationIndentationCheckTest

### PR DESCRIPTION
**Pull request by djydewang**
Issue #4586

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheckTest.java [hunk 1]
```diff
@@ -76,7 +79,8 @@ public void testCheck() throws Exception {
             "322: " + getCheckMessage(MSG_KEY, 4),
             "324: " + getCheckMessage(MSG_KEY, 4),
         };
-        verify(checkConfig, getPath("InputJavaDocTagContinuationIndentation.java"),
+        verify(checkConfig,
+                getPath("InputJavaDocTagContinuationIndentation.java"),
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheckTest.java]: File name didn't change so this line change is completely unnecessary.
Please undo change.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheckTest.java [hunk 2]
```diff
@@ -89,7 +93,8 @@ public void testCheckWithOffset3() throws Exception {
             "7: " + getCheckMessage(MSG_KEY, 3),
             "19: " + getCheckMessage(MSG_KEY, 3),
         };
-        verify(checkConfig, getPath("InputJavaDocTagContinuationIndentationOffset3.java"),
+        verify(checkConfig,
+                getPath("InputJavaDocTagContinuationIndentationOffset3.java"),
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheckTest.java]: same here.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoctagcontinuationindentation/InputJavadocTagContinuationIndentationGuavaFalsePositive.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package com.puppycrawl.tools.checkstyle.checks.javadoc;
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadoctagcontinuationindentation;
```
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoctagcontinuationindentation/InputJavadocTagContinuationIndentationGuavaFalsePositive.java]: Please rename class name to follow input's file name.

Comment (djydewang) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoctagcontinuationindentation/InputJavadocTagContinuationIndentationGuavaFalsePositive.java]: Fixed it.

**Comment by romani** [comment]
looks like all points are addressed

**Pull request merged**